content/pt-br/docs/concepts/configuration/windows-resource-management.md

```diff
@@ -15,29 +10,29 @@ Esta página descreve as diferenças em como os recursos são gerenciados entre
 
 <!-- body -->
 
-Em nós do Linux, {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} são usados como um limite de pod para controle de recursos. 
+Em nós Linux, {{< glossary_tooltip text="cgroups" term_id="cgroup" >}} são usados ​​como uma divisão para o controle de recursos em Pods. 
 Os contêineres são criados dentro desse limite para o isolamento de rede, processo e sistema de arquivos. 
-As APIs Linux cgroup podem ser usadas para coletar estatísticas de uso de CPU, E/S e memória.
+As APIs de cgroup do Linux podem ser usadas para coletar estatísticas de uso de CPU, E/S e memória.
 
-Em contraste, o Windows usa um [_objetos de trabalho_](https://docs.microsoft.com/windows/win32/procthread/job-objects) por contêiner com um filtro de namespace do sistema
+Em contraste, o Windows usa um [_objeto de trabalho_](https://docs.microsoft.com/windows/win32/procthread/job-objects) por contêiner com um filtro de namespace do sistema
 para conter todos os processos em um contêiner e fornecer isolamento lógico ao hospedar.
 (Os objetos de trabalho são um mecanismo de isolamento de processo do Windows e são diferentes dos
 que o Kubernetes chama de {{< glossary_tooltip term_id="job" text="Job" >}}).
 
 Não há como executar um contêiner do Windows sem a filtragem de namespace. 
-Isso significa que os privilégios do sistema não podem ser declarados no contexto do host e, 
+Isso significa que os privilégios do sistema não podem ser assegurados no contexto do host e, 
 portanto, os contêineres privilegiados não estão disponíveis no Windows.
-Os contêineres não podem assumir uma identidade do host porque o Gerente de conta de segurança (SAM) é separado.
+Os contêineres não podem assumir uma identidade do host porque o Gerente de Conta de Segurança ( Security Account Manager , ou SAM) é separado.
```

Comment:
nit: there's an excessive amount of spaces in "( Security Account Manager , ou SAM)". It's OK to not have spaces after the opening parentheses or before the comma, i.e. "(Security Account Manager, ou SAM)".